tajo-client/src/main/java/org/apache/tajo/client/CatalogAdminClientImpl.java

```diff
@@ -164,21 +166,22 @@ public TableDesc createExternalTable(final String tableName, final Schema schema
       throw new RuntimeException(e);
     }
 
-    if (isThisError(res.getState(), Errors.ResultCode.DUPLICATE_TABLE)) {
-      throw new DuplicateTableException(res.getState());
-    }
+    throwsIfThisError(res.getState(), DuplicateTableException.class);
+    throwsIfThisError(res.getState(), InsufficientPrivilegeException.class);
+    throwsIfThisError(res.getState(), UnavailableTableLocationException.class);
 
     ensureOk(res.getState());
     return CatalogUtil.newTableDesc(res.getTable());
   }
 
   @Override
-  public boolean dropTable(String tableName) throws UndefinedTableException {
-    return dropTable(tableName, false);
+  public void dropTable(String tableName) throws UndefinedTableException, InsufficientPrivilegeException {
+    dropTable(tableName, false);
   }
 
   @Override
-  public boolean dropTable(final String tableName, final boolean purge) throws UndefinedTableException {
+  public void dropTable(final String tableName, final boolean purge)
+      throws UndefinedTableException, InsufficientPrivilegeException {
```

Comment:
Similar to the case of the above ```createExternalTable```, these exceptions should be thrown properly at TajoMasterClientService.dropTable().